pkgs/top-level/all-packages.nix

```diff
@@ -5640,7 +5640,7 @@ with pkgs;
 
   waitron = callPackage ../tools/networking/waitron { };
 
-  pyCA = python3Packages.callPackage ../applications/video/pyca { };
+  pyCA = python3Packages.callPackage ../applications/video/pyca { inherit jq; };
```

Comment:
That inherit seems unnecessary?

Reply:
Yep, I thought so, too. But there is `python3Packages.jq` which will be used without this `inherit`

Reply:
Okay, fixed.